static/payment.js

```diff
@@ -0,0 +1,60 @@
+function setupPayment(stripeKey) {
+    var stripe = Stripe(stripeKey);
+    var elements = stripe.elements({locale: "sv"});
+
+    var card = elements.create('card', {
+        hidePostalCode: true,
+        style: {
+            base: {
+                iconColor: '#666EE8',
+                    <!-- color: '#31325F', -->
```

Comment:
That's not a JS comment... Also, I don't like mixing style and behaviour. But if it's the only way to customise Stripe, so be it. Is it possible to just use some sort of class selector instead?